archlinuxcn/v2raya-core/PKGBUILD

```diff
@@ -0,0 +1,14 @@
+# Maintainer: Rocket Aaron <i at rocka dot me>
+
+pkgname='v2raya-core'
+pkgdesc='v2ray core for v2rayA'
+pkgver=1
+pkgrel=1
+url='https://github.com/v2fly/v2ray-core'
+license=('MIT')
+depends=('v2ray')
+arch=('any')
+
+package() {
+    # nothing to package
+}
```

Comment:
This is bash...

Reply:
didn't know this before 🥲

should have wrote `:;` instead?

Reply:
Or `true`. `;` is not needed.